src/support/timefuncs.c

```diff
@@ -29,30 +29,31 @@ extern "C" {
 #endif
 
 #if defined(_OS_WINDOWS_)
-static double floattime(void)
+int gettimeofday(struct timeval *tv, void *tz)
 {
-    struct timeb tstruct;
-
-    ftime(&tstruct);
-    return (double)tstruct.time + (double)tstruct.millitm/1.0e3;
+    struct timeb tb;
+    ftime(&tb);
```

Comment:
Unfortunately the `struct timeval` on Windows only holds 32-bit values so their isn't much of a point in using `_ftime_64_s`. I have however switched to using `_ftime_s`